src/main/java/reposense/util/FileUtil.java

```diff
@@ -28,6 +32,9 @@
     private static Logger logger = LogsManager.getLogger(FileUtil.class);
 
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
+    private static final String TEMPLATE_ZIP_ADDRESS = "/templateZip.zip";
+    private static final String JSON_ZIP_FILE = "archiveJSON.zip";
```

Comment:
why is the zip and unzip folder name different?

Reply:
Both the methods `zipJson(..)` and `unzip(..)` are used on different files. 

`zipJson(...)` is used to zip all the generated json files and put them in a `archiveJSON.zip`; however, `unzip(...)` is used to unzip the `TEMPLATE_ZIP_ADDRESS` which contains all the html, css, js files.

Reply:
In that case, please add the comments. 
Especially those with specific purpose, used outside java code base.